### ISSUE TITLE
Per-network StatusPrefix

### ISSUE THREAD

**Issue by dreamcat4** [znc/znc#1242]
Hi,
I'm finding that each network opens it's own `*status` window in irssi client. So there are about 8 of them (1 for each different network), and they all look the same. This makes navigation to that network's status screen version very cumbersome.

They all look the same. So having to go into each window and read carefully looking out for a single word in the buffer, as to whether to dismiss or not. If unlucky, have to go through all 8 windows until get to the right one. Which usually takes ages.

I've asked on irssi people. But clearly also here is a good place to ask for per-network StatusPrefix. As that would also solve the problem too. Then each window could be labelled like: `*freenode_status`, `*efnet_status`, `*ircnet_status`, `*gnome_status`, etc.

**Comment by DarthGandalf** [comment]
When user switches between networks in the same connection, it can be confusing who he needs to speak to.

```
/msg *freenode_status jumpnetwork efnet
/msg *freenode_status help(oops)^W^W*efnet_status help
```
While speaking to the same ZNC

**Comment by DarthGandalf** [comment]
And there will be one more source of confusion.
There will need to be 2 different settings, one on user level, another one on network level.
The user level would be used in 2 situations: when client is connected to user, but not to network (it's very rarely, but possible), and when creating new network (just copy the value to the per-network setting).

Regardless of how the user setting will be presented, I expect users will confuse one with the other :(

**Comment by dreamcat4** [comment]
OK I think I understand now. 

Sorry I didn't quite get it before. For example eg `/msg -efnet *efnet_module` is rather redundant.

So we really need to be changing the way irssi client displays the windows, to include the network as a prefix somehow. Then indeed that is a client-side issue.

**Comment by dreamcat4** [comment]
However we it could be possible to have both a user + network level prefixes at same time. I mean to appear by concactenation. For example:

user-level Prefix: `$`
network-level Prefix: `efnet_`

full prefix: `$efnet_`

Perhaps from the programming side it would be more confusing to znc itself, having duplicate variables.

**Comment by Mikaela** [comment]
Couldn't irssi add the prefix by itself?

I use WeeChat with the buffers.pl script and I have `/set buffers.look.short_names off` so all IRC-buffers in my sidebar are in format `<network>.<channel|nick>`.

I name my networks by taking two first letters of the network name (or if that is not possible, as many until there aren't two same names) so my freenode \*status is `fr.*status` and oftc \*status is `of.*status` etc.

**Comment by dreamcat4** [comment]
@Mikaela yeah thanks. I agree it would be prefereable to do this on client side. Precisely what you have described, except for the irssi client instead of WeeChat. To be more specific: with irssi's `adv_windowlist.pl` channel bar.

Actually I would like to take things a step further, by selectively prefixing only for those channels named `*status` and `*module` etc. Leaving all regular IRC `#channel`s untouched.

**Comment by ailin-nemui** [comment]
you can enable network headers in adv_windowlist

**Comment by dreamcat4** [comment]
Hi @ailin-nemui,
Yes indeed but there is only a very limited amount of space. Only need it for some of the windows. Like with a regexp somehow to filter which ones have the prefix or not. Is that possible with adv_windowlist?

**Comment by ailin-nemui** [comment]
if you sort your windows by network you'll have one header for each network. anyway I believe this is off topic for znc

**Comment by dreamcat4** [comment]
> if you sort your windows by network

Sorry but I don't to do that. Am sorting with `chansort.pl`. @ailin-nemui Do you think I should open a new issue on irssi.irssi repo?
